src/Components/ResultList.tsx

```diff
@@ -22,7 +23,11 @@ export default class ResultList extends React.Component {
   constructor(props: any) {
     super(props);
 
-    this.headlessResultList = buildResultList(headlessEngine);
+    this.headlessResultList = buildResultList(headlessEngine, {
+      options: {
+        fieldsToInclude: ["date"]
```

Comment:
ah, interesting. Shouldn't we be explicitly including all fields we use in the interface then?

Those would be "source", "filetype", "date", and we might as well add "atlgeographicalhierarchy" right away

Reply:
ah I'll add `atlgeographicalhierarchy` 👍 
The other ones are default fields and there's no need to specify them :)

Reply:
ah actually I don't even think I'll include that one, since we don't use it in the result list.
The facet fields are requested by the facets themselves. The `fieldsToInclude` is really just for the Search API requests, in order for the result list to have all the fields it needs.